.docker/Viewer-v3.x/entrypoint.sh

```diff
@@ -7,9 +7,15 @@ if [ -n "$SSL_PORT" ]
     envsubst '${PORT}' < /usr/src/default.conf.template > /etc/nginx/conf.d/default.conf
 fi
 
-if [ -n "$APP_CONFIG" ]
+if [ -n "$APP_CONFIG" ] ; then
+  echo "$APP_CONFIG" > /usr/share/nginx/html${PUBLIC_URL}app-config.js
+  rm /usr/share/nginx/html${PUBLIC_URL}app-config.js.gz
+  gzip /usr/share/nginx/html${PUBLIC_URL}app-config.js
+  touch /usr/share/nginx/html${PUBLIC_URL}app-config.js
```

Comment:
can you add a comment why? so we put the APP_Config first via echo, then we remove and gzip again, so far makes sense. Why we are again creating an empty appConfig.js via `touch`?

Reply:
Because you have to ensure it gets compressed for serving it up.  That is done by having an empty file named app_config.js and a compressed file named app_config.js.gz